content/pt/docs/concepts/scheduling/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Escalonamento"
```

Comment:
Not sure Escalonamento is the right word to translate this. It would have more meaning on `Scaling`

What if we change it to `Agendamento`?

Reply:
Hi @femrtnz . I think that `Escalonamento` is better because this term is used in most  [pappers](https://scholar.google.com.br/scholar?hl=pt-BR&as_sdt=0%2C5&q=escalonamento+de+processos&oq=Escalonamento+) in Portuguese

For example, the abstract of [this one](https://www.redalyc.org/pdf/1331/133115038002.pdf) also use this term.